tests/unit/core/statistics/test_constraints.py

```diff
@@ -401,3 +408,49 @@ def test_max_between_constraint_invalid():
         maxBetweenConstraint(lower_value="2", upper_value=2)
     with pytest.raises(TypeError):
         maxBetweenConstraint(lower_field="max", upper_field=2)
+
+
+def test_column_values_in_set_constraint(df_lending_club, local_config_path):
+    cvisc = columnValuesInSetConstraint(value_set={2, 5, 8})
+    ltc = ValueConstraint(Op.LT, 1)
+    dc = DatasetConstraints(None, value_constraints={"annual_inc": [cvisc, ltc]})
+    config = load_config(local_config_path)
+    session = session_from_config(config)
+    profile = session.log_dataframe(df_lending_club, "test.data", constraints=dc)
+    session.close()
+    report = dc.report()
+
+    assert report[0][1][0][1] == len(df_lending_club)
+    assert report[0][1][1][1] == len(df_lending_club)
+
+
+def test_merge_values_in_set_constraint_different_value_set():
+    cvisc1 = columnValuesInSetConstraint(value_set={1, 2, 3})
+    cvisc2 = columnValuesInSetConstraint(value_set={3, 4, 5})
+    with pytest.raises(AssertionError):
+        cvisc1.merge(cvisc2)
+
+
+def test_merge_values_in_set_constraint_same_value_set():
+    cvisc1 = columnValuesInSetConstraint(value_set={"abc", "b", "c"})
+    cvisc2 = columnValuesInSetConstraint(value_set={"abc", "b", "c"})
+    merged = cvisc1.merge(cvisc2)
+    print(merged.report())
+
+
+def test_serialization_deserialization_values_in_set_constraint():
+    val_set = {"abc", 1, 2}
+    cvisc = columnValuesInSetConstraint(value_set=val_set)
+
+    cvisc.from_protobuf(cvisc.to_protobuf())
+    json_value = json.loads(message_to_json(cvisc.to_protobuf()))
+
+    print(json_value)
```

Comment:
remove `print`?

Reply:
Having debug/diagnostic output as part of a test is really useful, maybe replace the print statement with something like this:
https://github.com/whylabs/whylogs/blob/09e0289175ddcffb05a0e37953eed5e8347a1ca4/tests/component/test_performance_log_dataframe.py#L103